lib/boilerplate.js

```diff
@@ -0,0 +1,69 @@
+const { stripIndent } = require('common-tags');
+const util = require('./util');
+
+module.exports = function (moduleName, options = {}) {
+  return {
+    a2: {
+      'create-module': {
+        moduleConfig: stripIndent`
+          module.exports = {
+            extend: 'apostrophe-module',
+            label: '${util.titleCase(moduleName.replace(/-/g, ' '))}',
+            construct: function (self, options) {
+
+            }
+          };
+        `
+      },
+      'create-piece': {
+        pieceConfig: stripIndent`
+          module.exports = {
+            extend: 'apostrophe-pieces',
+            // TODO: Update the piece type name to be the singular version of the module
+            // name. For example, if the module is "people" the piece type name should be
+            // "person".
+            name: '${moduleName}',
+            // TODO: Update the singular label for editors
```

Comment:
I'm assuming there's no regression here.

Reply:
What would the regression be?